cmd/operator-sdk/alpha/bundle/cmd.go

```diff
@@ -0,0 +1,27 @@
+package bundle
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type bundleCmd struct {
+	directory      string
+	imageTag       string
+	imageBuilder   string
+	packageName    string
+	channels       []string
+	channelDefault string
+	overwrite      bool
+	generateOnly   bool
+}
+
+func NewCmd() *cobra.Command {
+	runCmd := &cobra.Command{
+		Use:   "bundle",
+		Short: "Operator bundle commands",
+		Long:  `Generate operator bundle metadata and build bundle image.`,
```

Comment:
I think it is missing info in the LONG description. 
What is the `bundle metadata`? What is the purpose? Why a user would like to do that? 
As describe what is the ` build bundle image` and/or add ref. links.